src/apps/loyalty/server/middlewares/user.ts

```diff
@@ -0,0 +1,21 @@
+import { NextFunction, Request, Response } from "express"
+import { fetchCollectorProfile } from "../gravity"
+
+export default (req: Request, res: Response, next: NextFunction) => {
```

Comment:
I think it's fair to move to res.locals once we remove `artsy-passport`'s dependency on backbone. It works pretty well for now.